tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/arena/PureArena.scala

```diff
@@ -0,0 +1,248 @@
+package at.forsyte.apalache.tla.bmcmt.arena
+
+import at.forsyte.apalache.tla.bmcmt.types.{CellT, UnknownT}
+import at.forsyte.apalache.tla.bmcmt.{ArenaCell, CheckerException}
+import at.forsyte.apalache.tla.lir.{NameEx, TlaEx}
+
+/**
+ * An SMT-context free implementation of arenas.
```

Comment:
This comment describes what `PureArena` is **not** doing. Shall we describe what it is doing too?

Reply:
I mean, `Arena`s (and the paper) already describe what arenas do. Why duplicate?